ArduCopter/GCS_Mavlink.cpp

```diff
@@ -993,6 +993,36 @@ void GCS_MAVLINK_Copter::handleMessage(mavlink_message_t* msg)
         break;
     }
 
+    case MAVLINK_MSG_ID_MANUAL_CONTROL:
+    {
+        if(msg->sysid != copter.g.sysid_my_gcs) break;                         // Only accept control from our gcs
```

Comment:
> 2017 and it is still in the top10 of mistakes found atfer refactoring ^^

That's because they don't have me doing code review :stuck_out_tongue: 

> it is in the ardupilot coding style 

Yes, but it doesn't need to be followed blindly, we can break it sometimes. Brackets here really doesn't add anything, you would be copying the exact same line elsewhere (being in the same line might actually prevent some copy/paste error).